sql/mozfun/norm/distribution_model_installs/udf.sql

```diff
@@ -0,0 +1,212 @@
+      )
+      THEN 'OEM onboarding'
+    WHEN distribution_id IN ('playanext-wt-001', 'playanext-wt-us-001')
+      THEN 'OEM onboarding'
+    WHEN distribution_id IN (
+        'isltd-g-001',
+        'isltd-g-aura-001',
+        'isltd-y-aura-001',
+        'isltd-y-001',
+        'isltd-001',
+        'isltd-002',
+        'isltd-003'
+      )
+      THEN 'OEM onboarding'
+    WHEN distribution_id IN ('MozillaOnline')
+      THEN 'partner website'
+    WHEN distribution_id IN (
+        'mozilla-win-eol-esr115',
+        'mozilla-mac-eol-esr1',
+        'mozilla-mac-eol-esr115'
+      )
+      THEN 'mozilla internal accounting'
+    WHEN distribution_id IN ('mozilla-EMEfree', 'EMEfree')
+      THEN 'Mozilla - EME free'
+    WHEN distribution_id IN (
+        'mozilla101',
+        'mozilla102',
+        'mozilla139',
+        'mozilla138',
+        'mozilla86',
+        'mozilla116',
+        'mozilla63',
+        'mozilla88',
+        'mozilla118',
+        'mozilla134',
+        'mozilla105',
+        'mozilla94',
+        'mozilla117',
+        'mozilla14',
+        'mozilla93',
+        'mozilla15',
+        'mozilla114',
+        'mozilla104',
+        'mozilla103',
+        'mozilla111',
+        'mozilla81',
+        'mozilla50',
+        'mozilla76',
+        'mozilla113',
+        'mozilla90',
+        'mozilla80',
+        'mozilla87',
+        'mozilla97',
+        'mozilla100',
+        'mozilla77',
+        'mozilla75',
+        'mozilla78',
+        'mozilla110',
+        'mozilla52',
+        'mozilla79',
+        'mozilla106',
+        'mozilla85',
+        'mozilla53',
+        'mozilla115',
+        'mozilla112',
+        'mozilla98',
+        'mozilla99',
+        'mozilla28',
+        'mozilla12',
+        'mozilla84',
+        'mozilla11',
+        'mozilla26',
+        'mozilla68',
+        'mozilla83',
+        'mozilla91',
+        'mozilla94-default',
+        'mozilla121',
+        'mozilla41',
+        'mozilla92',
+        'mozilla82',
+        'mozilla96',
+        'mozilla132',
+        'mozilla67',
+        'mozilla61',
+        'mozilla13',
+        'mozilla51',
+        'mozilla19',
+        'mozilla66',
+        'mozilla131',
+        'mozilla89',
+        'mozilla104-utility-existing',
+        'mozilla35',
+        'mozilla130',
+        'mozilla32',
+        'mozilla120',
+        'mozilla34',
+        'mozilla38',
+        'mozilla119',
+        'mozilla36',
+        'mozilla43',
+        'mozilla21',
+        'mozilla45',
+        'mozilla95',
+        'mozilla-cliqz-001',
+        'mozilla60',
+        'mozilla122',
+        'mozilla25',
+        'mozilla135',
+        'mozilla102-no-thanks',
+        'mozilla22',
+        'mozilla-cliqz-008',
+        'mozilla86-utility-existing',
+        'mozilla-cliqz-005',
+        'mozilla18',
+        'mozilla40',
+        'mozilla-cliqz-006'
+      )
+      THEN 'mozilla internal accounting'
+    WHEN (distribution_id IS NULL OR distribution_id != '0')
```

Comment:
@SuYoungHong should this be changed? right now it's saying if you gave some random new distribution ID like "ABCDEFG" it would go to "non-distribution" but I think we'd want it as "Uncategorized"?